### PR TITLE
Implement specs for books controller

### PR DESCRIPTION
**Pull request by sebasdeldi**
## Summary

- Implement specs for the index action of boooks controller

- Implement specs for the show action of boooks controller

## Trello Card

https://trello.com/c/mbrEOLGj/22-crear-tests-para-rutas-index-y-show-de-book

### CODE REVIEW

**Review by nicozar95 — APPROVED** [review]
Great work! 🎉 Beware of the EOF in the `.rubocop.yml`

File: spec/controllers/api/v1/books_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+RSpec.describe Api::V1::BooksController, type: :controller do
+  include_context 'Authenticated User'
+
+  describe 'GET #index' do
+    context 'When fetching all books' do
+      let!(:books) { create_list(:book, 3) }
+
+      before do
+        get :index, params: { id: user.id }
```
Comment (Gmarn) [inline spec/controllers/api/v1/books_controller_spec.rb:11]: There are project that use other convention, and try to define a `subject` for each `describe`

File: spec/controllers/api/v1/books_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
```
Comment (Gmarn) [inline spec/controllers/api/v1/books_controller_spec.rb:1]: Only this require should be here..
The other one should be import in your rails_helper, also ones from factories do not need to be import if you import `factory_bot` in the rails_helper

File: spec/shared_contexts/authenticated_user.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+shared_context 'Authenticated User' do
```
Comment (Gmarn) [inline spec/shared_contexts/authenticated_user.rb]: This should be in the directory `./spec/support/`

**Pull request merged**